### DISCUSSION TITLE
Multi data

### DISCUSSION THREAD

**Discussion by Luong2705** [elimintz/justpy#517] [Q&A]
I want multi data in map like that. But justpy not support 
![image](https://user-images.githubusercontent.com/63923038/189105908-1cd54d91-6ff4-4ee6-8903-caf06fb01b20.png)

**Comment by WolfgangFahl** [comment]
@Luong2705  You might want to ask this question at https://stackoverflow.com/questions/tagged/justpy and tag with justpy and other tags to to reach a bigger audience. justpy itself at this point has only 875 stars and 25 questions on stackoverflow while 
other solutions have a lot more stars and questions.
Stackoverflow also has much stricter rules than our discussions and issues. Please follow those stackoverflow rules to get better results for the community.
